### ISSUE TITLE
Panel App Invalid Evidences

### ISSUE THREAD

**Issue by ireneisdoomed** [opentargets/platform#1267]
After running the data pipeline for the 20/11 release 392 invalid evidences have been identified for the data source Genomics England.
These strings correspond to 119 source diseases that have not passed the validation process because they could not be associated with any EFO term.

An analysis of each of the terms can be found here: https://docs.google.com/spreadsheets/d/1CsDgIm3yaMH28IIgMIDKqstbonAwTpb1b_wjE114NW4/edit?usp=sharing

Following the analysis, a series of steps are set out to be completed:
- [X] Open ticket to EFO for inclusion of 93 MONDO terms. See [here](https://github.com/EBISPOT/efo/issues/894)
- [X] Open ticket to EFO for the addition of 9 synonym to existing EFO terms. See [here](https://github.com/EBISPOT/efo/issues/895).
- [X] Update manual mapping file with the found errors. 
- [ ] Improve programmatically the mapping of OnToma when consulting the OBO file.

For some of these actions, we are awaiting the new EFO release to see the outcome and push the changes.

**Comment by ireneisdoomed** [comment]
Update: new EFO release is due on December 15th and these changes are expected to be implemented.

**Comment by ireneisdoomed** [comment]
The terms changed by EFO in the December release have been reviewed and practically all are mapped correctly by OnToma.
- There are 4 cases in which the MONDO ID has been included to EFO but the term described in Panel App is not as synonymous. A new issue is opened to EFO to resolve this.
- MONDO has been asked to add several synonyms. [This issue](https://github.com/monarch-initiative/mondo/issues/2461#issuecomment-746905418) is still open because of an open discussion on their side.
- Although the mapping should already be correct programmatically, the [manual mappings file](https://github.com/opentargets/platform_semantic/blob/master/resources/zooma/cttv_indications_3.txt) has been updated so that there are no discrepancies.
- The improvement of the OnToma tool is still pending, that issue is described here [#1306](https://github.com/opentargets/platform/issues/1306).

Also to note: regarding the cases of terms tagged as `Gene with protein product`, we've opened [an issue](https://github.com/EBISPOT/efo/issues/914) to EFO for them to know if they can be imported somehow but given they are not very informative in terms of disease description they have been left for further discussion.
